src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
Works for me.  I 'll make the changes tonight.  BTW we are broken again in the just released 1.0.0 alpha.  If you try to start DebugTest.ps1 under the debugger with no breakpoints set, you get no output.  :-(